src/CommonITILObject.php

```diff
@@ -6518,6 +6518,17 @@ public function getTimelineItemtypes(): array
                 'item'      => $validation
             ];
         }
+        if ($canadd_document) {
+            $itemtypes['document'] = [
+                'type'          => 'Document_Item',
+                'class'         => Document_Item::class,
+                'icon'          => Document_Item::getIcon(),
+                'label'         => _x('button', 'Add a document'),
+                'template'      => 'components/itilobject/timeline/form_document_item.html.twig',
+                'item'          => new Document_Item(),
+                'show_in_menu'  => false
```

Comment:
```suggestion
                'hide_in_menu'  => true
```

Reply:
Your last commit reverted change i mage in the previous one. `show_in_menu` is not used anymore.